src/Make_mvc.mak

```diff
@@ -1312,7 +1312,7 @@ notags:
 
 clean: testclean
 	- if exist $(OUTDIR)/nul $(DEL_TREE) $(OUTDIR)
-	- if exist *.obj del *.obj
+	- if exist $(OUTDIR)\*.obj del $(OUTDIR)\*.obj
```

Comment:
I think this is correct.

`$(OUTDIR)\*.obj` will be deleted by `$(DEL_TREE)` command in the previous line.
Some .obj files will be created in current directory, so `del *.obj` is needed.

Reply:
Hmm, ok. I think the issue is that `$(OUTDIR)/nul` is not created. At least when I'm building locally it doesn't seem to be created and as such the `$(DEL_TREE)` never gets called. Do you think it's some relic that got removed?

Reply:
Oh, `$(OUTDIR)/nul` should be `$(OUTDIR)\nul`.
`dirname\nul` can be used to check the existence of `dirname` directory.

Reply:
Hmm, something must be wrong either with the make file or my environment then (using VS 2022's developer tools). I could take a quick look.

Reply:
Actually I think you are right. I probably must have made a mistake when I was looking into this. Let me close this. Thanks for sanity checking this.